ignite/handlers/checkpoint.py

```diff
@@ -401,6 +403,14 @@ def __call__(self, checkpoint: Mapping, filename: str, metadata: Optional[Mappin
                 tmp.close()
                 os.rename(tmp.name, path)
 
+    def _save(self, checkpoint: Mapping, filename: str):
+        if idist.has_xla_support:
```

Comment:
It could be in models ?

Reply:
Which models you mean ?

Reply:
CompModels. Like barrier, reduction. Maybe not a good idea.

Reply:
Well, I'm not sure that `idist` should replace all possible ops. IMO, specific code should be specific...